free-programming-books.md

```diff
@@ -1029,7 +1028,7 @@ Original Source: [List of freely available programming books](http://web.archive
 * [Developer Workbooks](http://developer.force.com/workbook)
 * [Force.com Fundamentals](https://developer.salesforce.com/page/Force_Platform_Fundamentals)
 * [Force.com Platform Fundamentals: An Introduction to Custom Application Development in the Cloud](http://www.lulu.com/shop/salesforcecom/forcecom-platform-fundamentals/ebook/product-17381451.html)
-* [Heroku Postgres](http://media.developerforce.com/workbooks/HerokuPostgres_Workbooks_Web_Final.pdf) (PDF)
+* [Heroku Postgres](http://www.scribd.com/doc/256342154/herokupostgres-workbooks-web-final-13-pdf) (PDF)
```

Comment:
@vhf what are your thoughts on links from Scribd? I have visited the site, but not in a while, so am mainly unfamiliar with the content and licensing.